src/mumble/AudioInput.cpp

```diff
@@ -1048,6 +1048,11 @@ void AudioInput::encodeAudioFrame(AudioChunk chunk) {
 				if (!qetLastMuteCue.isValid() || qetLastMuteCue.elapsed() > MUTE_CUE_DELAY) {
 					qetLastMuteCue.start();
 					ao->playSample(Global::get().s.qsTxMuteCue, Global::get().s.cueVolume);
+
+					if (!Global::get().s.bMuteCueShown && !Global::get().bInConfigUI) {
+						Global::get().mw->initializeAIConnection();
```

Comment:
The connection of signal and slot should usually happen once in the constructor. See [this](https://github.com/mumble-voip/mumble/blob/master/src/mumble/AudioInput.cpp#L288) for an example similar to the new signal slot connection.